source/retryable-writes/tests/insertOne.yml

```diff
@@ -53,3 +53,58 @@ tests:
                 data:
                     - { _id: 1, x: 11 }
                     - { _id: 2, x: 22 }
+    -
+        description: "InsertOne succeeds after InterruptedDueToReplStateChange"
+        failPoint:
```

Comment:
I might suggest leaving this `failPoint` key as-is and just adding `configureFailPoint: NAME` as the first key. This will effectively allow drivers to use the `failPoint` document in the test case as the command document directly.

Reply:
Thanks SGTM!